extras/iris_pipeline_project/data_folder/iris_train.csv

```diff
@@ -0,0 +1,508 @@
+SepalLength,SepalWidth,PetalLength,PetalWidth,Species
```

Comment:
I guess we don't need that in the repo since we have an instruction on preparing the data.